snapshots/countries/VN_COMMON.json

```diff
@@ -508,7 +508,7 @@
     "2009-01-27": "Second Day of Lunar New Year",
     "2009-01-28": "Third Day of Lunar New Year",
     "2009-01-29": "Fourth Day of Lunar New Year",
-    "2009-04-05": "Hung Kings' Commemoration Day",
+    "2009-04-04": "Hung Kings' Commemoration Day",
```

Comment:
This is indeed correct 👀👍
 https://english.bvhttdl.gov.vn/articledetail.aspx?articleid=22694&sitepageid=415